### ISSUE TITLE
Opening new Atom window changes desktop / workspace

### ISSUE THREAD

**Issue by jchook** [atom/atom#18691]
### Prerequisites

* [x] Put an X between the brackets on this line if you have done all of the following

### Description

Opening new Atom windows switches the focused desktop / workspace.

#### Related:

* [New Window Changes Desktop Assignment of Other Windows](https://discuss.atom.io/t/new-window-changes-desktop-assignment-of-other-windows/40999/6)

### Steps to Reproduce

1. Open Atom on workspace 1
2. Switch to workspace 2
3. Open new atom window on workspace 2

**Expected behavior:**

Atom window opens on workspace 2. Window manager remains on workspace 2.

**Actual behavior:**

* Atom changes window manager focus back to workspace 1
* New atom window opens on workspace 1

**Reproduces how often:** 100%

### Versions

```
➜ atom --version
Atom    : 1.34.0
Electron: 2.0.16
Chrome  : 61.0.3163.100
Node    : 8.9.3

➜ apm --version
apm  2.1.3
npm  6.2.0
node 8.9.3 x64
atom 1.34.0
python 2.7.15rc1
git 2.17.1

➜ cat /etc/*release
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=18.04
DISTRIB_CODENAME=bionic
DISTRIB_DESCRIPTION="Ubuntu 18.04.1 LTS"
NAME="Ubuntu"
VERSION="18.04.1 LTS (Bionic Beaver)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 18.04.1 LTS"
VERSION_ID="18.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=bionic
UBUNTU_CODENAME=bionic

➜ xmonad --version
xmonad 0.13

```

### Additional Information

Atom will open new windows on the desktop / workspace where the first atom window was opened.

For example, if I open Atom first on workspace 4, then switch to workspace 2, then open a new Atom window, it jumps back to workspace 4.

Only months ago, Atom did not behave this way.

**Comment by jchook** [comment]
Hey @rsese I appreciate your kind close and link to related issue.

This issue is patently different. Please re-open.

In the issue you linked, the problem is quite different. The user expects both files to be opened in the same atom instance.

In my case, I just want to fix some bad regression that disrupts my workflow.

**Comment by rsese** [comment]
Yes the issues aren't exactly the same but are both related to what workspace a window opens in which is related to the referenced Electron issue: https://github.com/electron/electron/issues/5362. 
 E.g. see https://github.com/atom/atom/issues/13766 which was closed in favor of https://github.com/electron/electron/issues/5362 and is closer to the issue you describe but also different than https://github.com/atom/atom/issues/11610.

**Comment by jaller94** [comment]
Does this really require a workspace API in Electron?
I am not looking for Atom to manage my workspaces.

This issue is that the first Atom window gets raised, when opening a nth Atom window.

In XFCE the resulting behaviour can be controlled with "When a window raises itself:" in the "Focus" tab of the "Window manager tweaks". Even setting it to "Do nothing" will cause it to blink in the window button list.
![xfce-focus](https://user-images.githubusercontent.com/10872136/55132769-b0c35b00-50e0-11e9-864a-6ada4a5c8e20.png)

**Comment by jchook** [comment]
@rsese can you verify this?

> This issue is that the first Atom window gets raised, when opening a nth Atom window.

Does this mean we can re-open and fix this ticket separately from the longer term Workspace API endeavor?

**Comment by jchook** [comment]
**Update** 

Issue fixed for me as of this version:

```
Atom    : 1.40.1
Electron: 3.1.10
Chrome  : 66.0.3359.181
Node    : 10.2.0
```
